docs/contribute/index.qmd

```diff
@@ -35,7 +35,7 @@ First, set up pixi as described on their getting started [page](https://prefix.d
 Then set up the environment by running the following commands:
 
 ```sh
-pixi run install
+pixi run --environment=dev install
```

Comment:
This is only necessary because otherwise the editable install installs into the wrong environment.
As soon as pixi supports editable installs this will not be necessary anymore.

Reply:
Perhaps it's good to create an issue for this, with a reminder to remove this.